hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/DatanodeStateMachine.java

```diff
@@ -125,7 +125,7 @@ public DatanodeStateMachine(DatanodeDetails datanodeDetails,
         .addHandler(new DeleteBlocksCommandHandler(container.getContainerSet(),
             conf))
         .addHandler(new ReplicateContainerCommandHandler(conf, supervisor))
-        .addHandler(new DeleteContainerCommandHandler())
+        .addHandler(new DeleteContainerCommandHandler(2))
```

Comment:
Can you define this as delete thread pool Size somewhere, otherwise 2 is kind of confusing to people.

Reply:
Yes, that "2" needs to be configurable just like for the "replicate container thread pool size" in HDDS-2450. Now HDDS-2450 has been committed, I will rebase this patch and make this thread pool configurable in the same way.